exorelay-go/bin/lint

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+golint
```

Comment:
Yes, let's change the import path to that. One would clone this repo into $GO_PATH/src/github.com/Originate/exocom. This is not a bad setup for coding anyways.